src/servers/launcher/launcher-review-request.md

```diff
@@ -0,0 +1,82 @@
+# Description
+
+For fun, and to learn some C++, I've decided to reimplement the entirety of EVE online as I remember it.  (I'm aware this is an insane and likely unreachable goal, that's intentional).
```

Comment:
This has to be the coolest and most ambitious learning project I have ever seen. Simply badass.